master/usage/calicoctl/configure/kdd.md

```diff
@@ -46,17 +46,17 @@ spec:
 #### Example using environment variables
 
 ```shell
-$ export DATASTORE_TYPE=kubernetes
-$ export KUBECONFIG=~/.kube/config
-$ calicoctl get workloadendpoints
+export DATASTORE_TYPE=kubernetes \
+export KUBECONFIG=~/.kube/config \
+calicoctl get workloadendpoints
 ```
 
 And using `CALICO_` prefixed names:
 
 ```shell
-$ export CALICO_DATASTORE_TYPE=kubernetes
-$ export CALICO_KUBECONFIG=~/.kube/config
-$ calicoctl get workloadendpoints
+export CALICO_DATASTORE_TYPE=kubernetes \
```

Comment:
I don't think this is going to work - this appends the lines together so that they are executed as a single command.

You'll need to separate the commands, or add `&&` between them to chain the commands together.

Reply:
This goes for other instances of this change in this PR as well.